docs/how-to-guides/data-basics/4-building-a-create-page-form.md

```diff
@@ -0,0 +1,373 @@
+# Part 4: Building a _Create page_ form
+
+In the [previous part](/docs/how-to-guides/data-basics/3-building-an-edit-form.md) we created an *Edit page* feature, and in this part we will add a *Create page* feature. Here's a glimpse of what we're going to build:
+
+![](/docs/how-to-guides/data-basics/media/create-form/create-form-with-text.png)
+
+### Step 1: Add a _Create a new page_ button
+
+Let’s start by building a button to display the _create page_ form. It’s similar to an _Edit_ button we have built in the [part 3](/docs/how-to-guides/data-basics/3-building-an-edit-form.md):
+
+```js
+import { useDispatch } from '@wordpress/data';
+import { Button, Modal, TextControl } from '@wordpress/components';
+
+function PageCreateButton() {
+	const [ isOpen, setOpen ] = useState( false );
+	const openModal = () => setOpen( true );
+	const closeModal = () => setOpen( false );
+	return (
+		<>
+			<Button onClick={ openModal } variant="primary">
+				Create a new Page
```

Comment:
I know we don't need translation functions here but is it better to use them to show the best practice?

Reply:
Let's add that to all parts in a follow-up PR to avoid suddenly introducing translations in part 4.